README.md

```diff
@@ -1 +1,12 @@
-# Wayscript TinyDB Template
+# TinyDB API Lair template
+
+A simple [TinyDB](https://tinydb.readthedocs.io/en/latest/) API Lair template that can be used to provision databases quickly.
+
+## Setup
+
+To start using this API -
+
+1. Create a new WayScript Lair from this template
+2. Deploy lair by running the pre-configured deploy trigger

```

Comment:
Since this is for storage, we should add guidance that they should use this lair via the dev endpoint, so deploys don't overwrite the data